### PR TITLE
add usedRukMaiChaiDuangDaoTeePrawSang

### PR DESCRIPTION
**Pull request by nonkung51**
- added usedRukMaiChaiDuangDaoTeePrawSang

this hook just added `รักไม่ใช่ดวงดาวที่พราวแสง` after any input

### CODE REVIEW

**Review by narze — CHANGES_REQUESTED** [review]
Looking good, but it should be `use`RukMaiChaiDuangDaoTeePrawSang instead of `used`RukMaiChaiDuangDaoTeePrawSang, could you change them?

**Comment by nonkung51** [comment]
Sure! Really sorry i changed it already na krub

File: src/hooks-usage/useRukMaiChaiDuangDaoTeePrawSang.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+// Default export a functional component which includes hook's usage
+// You can use our Card component or roll your own container component
+export default function () {
+  const cardProps = {
+    desc: "useRukMaiChaiDuangDaoTeePrawSang - returns something yada yada yada",
+    examples: [
+      // 1.) For simple form, just pass the hook's return value
+      {
+        code: 'const value = usedRukMaiChaiDuangDaoTeePrawSang("จาวา ไม่ใช่ จาวาสคริปต์")',
```
Comment (narze) [inline src/hooks-usage/useRukMaiChaiDuangDaoTeePrawSang.tsx]:
```suggestion
        code: 'const value = useRukMaiChaiDuangDaoTeePrawSang("จาวา ไม่ใช่ จาวาสคริปต์")',
```

**Review by narze — COMMENTED** [review]
👍🏻

**Pull request merged**